### ISSUE TITLE
Confirm if this is the official source code repo

### ISSUE THREAD

**Issue by chenrui333** [opencor/opencor#2501]
👋  Just curious what is the relationship between this repo and the site, https://www.riverbankcomputing.com/software/qscintilla/intro. Is this official source code repo or mirror? Thanks!

cc @agarny

**Comment by agarny** [comment]
This repository (https://github.com/opencor/opencor) is for a modelling environment called [OpenCOR](https://opencor.ws/) which uses [QScintilla](https://riverbankcomputing.com/software/qscintilla/intro) for its text-based editing capabilities.

Now, I imagine you are talking about another repository (https://github.com/opencor/qscintilla) which is also under the [OpenCOR organisation](https://github.com/opencor). That repository contains a copy (not mirror) of the QScintilla codebase, together with changes that are needed to build QScintilla as part of our overall OpenCOR project.

Bottom line: https://github.com/opencor/qscintilla is not the offical repository for QScintilla, or even a mirror of it.

**Comment by chenrui333** [comment]
I see, that makes sense. Thanks @agarny!
